jumeg/connectivity/con_viz.py

```diff
@@ -260,7 +260,14 @@ def plot_connectivity_circle(con, node_names, indices=None, n_lines=None,
         if con.shape[0] != n_nodes or con.shape[1] != n_nodes:
             raise ValueError('con has to be 1D or a square matrix')
         # we use the lower-triangular part
-        indices = np.tril_indices(n_nodes, -1)
+
+        is_symmetric = np.all(np.abs(con - con.T) < 1e-8)
+        if is_symmetric:
+            indices = np.tril_indices(n_nodes, -1)
+        else:
+            # get off-diagonal indices
+            indices = np.where(~np.eye(con.shape[0], dtype=bool))
```

Comment:
If the matrix is assymmetric, set arrow=True so that a causality matrix is automatically assumed. Maybe including a Warning / Info would also be useful to avoid confusion.